karina-lab10/public/scripts/articleView.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+var app = app || {module};
+
+var articleView = {module};
+
+
+(function (module) {
+
```

Comment:
var articleView = {};

we just want to instantiate an articleView object that we can attach properties to